### ISSUE TITLE
How to give a custom voice in as speaker? 

### ISSUE THREAD

**Issue by fatima-falconit** [myshell-ai/MeloTTS#172]
the model is working perfectly on predefined speaker ids but I wanna give my own voice for cloning. How can I do that?

**Comment by meet1919** [comment]
checkout this [https://github.com/myshell-ai/OpenVoice](url)

**Comment by jidkano** [comment]
@meet1919 this seems to be a completely different model that is nowhere near as fast as MeloTTS

**Comment by yukiarimo** [comment]
You can fine-tune MeloTTS

**Comment by rushichavda** [comment]
@yukiarimo do you have any resources/code which can help in finetuning? i have 1 hour of english data, and want to add new voice. Thanks

**Comment by yukiarimo** [comment]
Yes, I'll drop some stuff soon. Prepare you're dataset to be 3-30s chunked audio with transcripts for each.
